### DISCUSSION TITLE
［求助贴］最痛苦的一集！

### DISCUSSION THREAD

**Discussion by aminoc12138** [share121/inter-knot#1107] [其他]
已经不能正常进食了，口腔溃疡+牙龈发炎，准备明天去拔智齿，但现在好饿，好想吃点东西，怎么办！
![Uploading 1720891254891.png…]()

**Comment by zzzuser46** [comment]
没事，我替你吃好吃的  -_-

**Comment by taci1101** [comment]
不是应该等发炎期过再拔牙吗？我牙疼的时候就吃止痛药，饿的话就喝粥
